house_of_refuge/main/models.py

```diff
@@ -39,11 +39,11 @@ class TransportRange(models.TextChoices):
 
 
 class Status(models.TextChoices):
-    NEW = "new", _("Świeżak")
-    TAKEN = "taken", _("Zajęta")
-    CALLING = "calling", _("Dzwonimy")
-    IGNORE = "ignore", _("Ignoruj")
-    SHOULD_DELETE = "should_delete", _("Do usunięcia")
+    NEW = "new", _("Fresh")
```

Comment:
maybe let's keep it as `New` in the translatable string

Reply:
Ok, but then we should probably also use "Nowy" instead of "Świeżak" in Polish. Translating "New" to "Świeżak" wouldn't be great because than gettext would want to use "Świeżak" for all current and future occurrences of "new" in the system.

Reply:
you're right. I think I'm fine with that.